Slim/Networking/Async/HTTP.pm

```diff
@@ -43,6 +43,7 @@ use URI;
 use File::Spec::Functions qw(catdir);
 
 use Slim::Networking::Async::Socket::HTTP;
+use Slim::Networking::Async::Socket::HTTPsocks;
```

Comment:
Now this is the most nitpicky detail I could find: I'd use `HTTPSocks` :grin:.

Another small change: could please not `use` this module, but `require` it only when needed? Eg. inside the `elsif ($self->socksAddr) ...` conditional code.

Reply:
I was hesitating on that (HTTPSocks), but I'll do as you say. I'll move to a require as well